Rahulsingh210345.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "Rahulsingh210345"
+project: "Smart Plug"
+---
+
+# Smart Plug
+
+## Summary
+I am going to build a smart plug.This device will  allow users to remotely control and schedule the power to an electrical outlet through a smartphone app or voice assistant.this smart plug is way to learn about the inner workings of smart home technology and gain hands-on experience with electrical engineering and programming.
+
+ 
+## Plan
+To create a Smart Plug following  using the specified components, I will follow these steps:
+
+First I will make the Smart plug with remote monitoring (current, voltage, energy), control (automatic, manual), and notification.then I will connect wifi through appliances remotely. No matter where you are, integrate your device with the web server Then I will attach the Espressif ESP8266 ESP-01ESP WiFi Module.Then Set the Schedules/Timer  for turning ON and OFF your devices automatically,saving power and Time.This smart plug helps you monitor your devices energy consumption for the entire day and is suitable for conecting electrical appliances like Geysers,motors for Water Tanks,Heaters,Microwave Ovens,ACs,etc
+
+|[ Product                      | Supplier/Link                         | Cost    |
+| ---------------------------- | ------------------------------------- | ------- |
+| ESP Wifi Module              | https://www.sparkfun.com/products/17146          | $30.00  |
+| AC to DC Converter           |  https://www.amazon.com/Transformer-MKASYON-Switching-Converter-Security/dp/B07GFLFTV2     | $23.00|
+| Measuring Voltage and Current|  https://www.tme.eu/en/details/abx00041/arduino-solutions/arduino/nano-motor-carrier/      | $14.00 |
+| Smart Wi-Fi Plug Mechanical Box |https://jujukart.com/shop/smart-electronics/teckin-smart-wi-fi-plug-with-price-in-nepal/| $40.00 |
+|Arduino Nano Motor Carrier      | https://store.arduino.cc/products/arduino-nano-motor-carrier  |$95.80|
```

Comment:
What's the plan for the motor carrier?

Reply:
Extra  demand for another project

Reply:
@Rahulsingh210345 what do you mean by extra demand for another project?

Reply:
Then what do I do?

Reply:
> @Rahulsingh210345 what do you mean by extra demand for another project?